SnoopCompileBot/src/snoop_bench.jl

```diff
@@ -1,37 +1,29 @@
 # Snooping functions
 function _snoopi_bench_cmd(snoop_script)
-    # quote end generates $ which doesn't work in commands
-    # TODO no escape is done for snoop_script!!!
-    # TODO
-    # data = Core.eval(  $test_modul, _snoopi($(esc(snoop_script)))  )
-    # TODO use code directly for now
-    # no filter in the benchmark
-    return """
-        using SnoopCompile
+    tmin = 0.0 # For benchmarking
+    return quote
         global SnoopCompile_ENV = true
 
-        empty!(SnoopCompile.__inf_timing__)
-        SnoopCompile.start_timing()
-        try
-            $(string(snoop_script));
-        finally
-            SnoopCompile.stop_timing()
+        using SnoopCompileCore
+
+        data = @snoopi tmin=$tmin begin
+            $snoop_script
```

Comment:
Much better this way, I should have noticed this earlier.

Reply:
It was like this originally, but because of those macro issues, I wanted to use `_snoopi` instead, which was not working at the time, and so I switched to direct code. 

Now I wanted to try this, I get `@snoopi` not defined. Does this work for you?